scalableminds/webknossos
# Make jobs readable for admins+teamManagers+superusers

The job list should still only show the jobs of the requesting user. But if an admin or teamManager or superUser hast the id/link to a job by one of their managed users, they should be able to view its properties (GET api/jobs/:id) and access exported files.

### Steps to test:
- Enable jobs (`yarn enable-jobs`), set up different user/admin combinations
- As the user, run some jobs
- Check that everyone can see exactly those jobs they are supposed to see
- Also check export download links (for this you need a webknossos-worker. I did test it, though, and this uses the same access check as the simple GET api/jobs/:id, which can be tested without a worker)

### Issues:
- fixes #7447

------
